mutacc_auto/commands/mutacc_command.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, config_file, padding, case_file):
         super(MutAccExract, self).__init__(config_file)
 
         self.add_subcommand('extract')
-        self.add_option('padding', value=padding)
+        self.add_option('padding', value=str(padding))
```

Comment:
Why is padding a string and not an int

Reply:
Converts to string, the complete command line command will ultimately be given as a string, i.e. 'mutacc --config-file ... extract --padding 123 --case ...'